### PR TITLE
fixes default validity order time bug

### PR DESCRIPTION
**Pull request by W3stside**
1. Order validity time default. Can be set in Trade Widgets

Defaults to 2880 (2 days)
Maybe best to move Order form defaults somewhere else

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr550--dexreact.review.gnosisdev.com

File: src/components/TradeWidget/index.tsx [hunk 1]
```diff
@@ -71,10 +71,10 @@ export type TradeFormData = {
   [K in keyof typeof TradeFormTokenId]: string
 }
 
-const DEFAULT_FORM_STATE = {
+export const DEFAULT_FORM_STATE = {
   sellToken: '0',
   receiveToken: '0',
-  validUntil: '0',
+  validUntil: '30',
```
Comment (anxolin) [inline src/components/TradeWidget/index.tsx]: 2880 or 30? I'm confused. I'm slightly more inclined for `2880`. There was a tie in the vote

Comment (W3stside) [reply src/components/TradeWidget/index.tsx]: woops didnt push

**Comment by W3stside** [comment]
Closing for hotfix version

**Pull request closed without merging**